pages/api/v1/user-created-bounties/createBounty.ts

```
@@ -0,0 +1,132 @@
+			}
+			if (!proposerAddress?.length || !getEncodedAddress(proposerAddress, network)) {
+				return res.status(400).json({ message: 'Invalid Proposer Address.' });
+			}
+			if (!firebaseFormatedDeadline) {
+				return res.status(400).json({ message: 'Invalid Deadline Date.' });
+			}
+
+			if (!title?.length || !content?.length || isContentBlacklisted(content)) {
+				return res.status(400).json({ message: 'Title or Content is Missing or Invalid in request body.' });
+			}
+			if (isNaN(maxClaim) || !maxClaim) {
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)